sashimi-webapp/src/logic/filemanager/data/file.js

```diff
@@ -1,13 +1,12 @@
 import storage from 'src/database/storage';
 import Folder from './folder';
+import idMap from './idmap';
 
-/* Constants */
-
-// const RENAME_ERROR_MSG = `Another file in "${this.parentFolder.path}" has the same file name`;
-const RENAME_ERROR_MSG = 'Another file in the current folder has the same name';
-const MOVE_SAME_FOLDER_ERROR_MSG = 'Attempting to move to current folder';
-const MOVE_INVALID_FODLER_ERROR_MSG = 'Attempting to move to an invalid folder';
-const NOT_FILE_INSTANCE_ERROR_MSG = '"this" is not an instance of "File"';
+/* Error Messages */
+const ERROR_RENAME_FILE = 'Another file in the current folder has the same name';
```

Comment:
May want to rename to 
`ERROR_SAME_FILE_NAME_EXISTS` or 
`ERROR_FILE_NAME_ALREADY_EXISTS` 
to make variable name more meaningful